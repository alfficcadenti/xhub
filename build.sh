#!/bin/bash

set -e

printBoxedMessage()
{
  echo
  echo "--------------------"
  echo $1
  echo "--------------------"
}

printBoxedMessage "Print Build Environment"
echo "***** Node Version:"
node -v
echo "***** npm version:"
npm -v

###
# Install Dependencies
###
printBoxedMessage "Install"
npm install

###
# Run package building
###
printBoxedMessage "Build"
npm run build

###
# Run unit & linting tests
###
printBoxedMessage "Test"
npm test

###
# Run prune
###
printBoxedMessage "Prune"
npm prune --production

###
# Run dedupe
###
printBoxedMessage "Dedupe"

# if dedupe fails, it is probably due to this: https://github.com/npm/npm/issues/20173
# to remedy: check in package-lock.json and remove the "|| printBoxedMessage ..."
npm dedupe # || printBoxedMessage "WARNING: dedupe failed, ignoring for now"
