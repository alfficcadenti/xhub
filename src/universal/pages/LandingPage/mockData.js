// export const CSRData = [
//     {brandName: 'BEX', CSRTrend: [1, 2, 3, 6, 7, 3, 4, 10], CSRByDevice: {mobile: 90, tablet: 99, desktop: 60}},
//     {brandName: 'Hotels.com', CSRTrend: [1, 2, 3, 6, 5, 11, 10, 6], CSRByDevice: {mobile: 10, tablet: 99, desktop: 60}},
//     {brandName: 'Vrbo', CSRTrend: [10, 9, 3, 4, 5, 6, 4, 9], CSRByDevice: {mobile: 97, tablet: 50, desktop: 60}},
// ];

export const CSRData = [
    {
        'time': '2020-05-21T10:30:00.000Z', BEX: 0.99, Vrbo: 0.1, 'Hotels.com': 0.1,
    },
    {
        'time': '2020-05-21T11:30:00.000Z', BEX: 0.9, Vrbo: 0.1, 'Hotels.com': 0.2,
    },
    {
        'time': '2020-05-21T12:30:00.000Z', BEX: 0.9, Vrbo: 0.1, 'Hotels.com': 0.3,
    },
    {
        'time': '2020-05-21T13:30:00.000Z', BEX: 0.9, Vrbo: 1, 'Hotels.com': 0.7,
    },
    {
        'time': '2020-05-21T14:30:00.000Z', BEX: 0.9, Vrbo: 0.9, 'Hotels.com': 0.98,
    },
    {
        'time': '2020-05-21T15:30:00.000Z', BEX: 0.5, Vrbo: 0.7, 'Hotels.com': 0.8,
    },
    {
        'time': '2020-05-21T16:30:00.000Z', BEX: 0.99, Vrbo: 0.89, 'Hotels.com': 0.72,
    },
];

export const bookingsData = [
    {
        'time': '2020-05-21T10:30:00.000Z', BEX: 4000, Vrbo: 2400, 'Hotels.com': 2400,
    },
    {
        'time': '2020-05-21T10:30:00.000Z', BEX: 3000, Vrbo: 1398, 'Hotels.com': 2210,
    },
    {
        'time': '2020-05-21T10:30:00.000Z', BEX: 2000, Vrbo: 9800, 'Hotels.com': 2290,
    },
    {
        'time': '2020-05-21T10:30:00.000Z', BEX: 2780, Vrbo: 3908, 'Hotels.com': 2000,
    },
    {
        'time': '2020-05-21T10:30:00.000Z', BEX: 1890, Vrbo: 4800, 'Hotels.com': 2181,
    },
    {
        'time': '2020-05-21T10:30:00.000Z', BEX: 2390, Vrbo: 3800, 'Hotels.com': 2500,
    },
    {
        'time': '2020-05-21T10:30:00.000Z', BEX: 3490, Vrbo: 4300, 'Hotels.com': 2100,
    },
];