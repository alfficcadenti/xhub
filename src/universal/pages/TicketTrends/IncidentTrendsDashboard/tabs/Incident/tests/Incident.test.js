import React from 'react';
import {shallow} from 'enzyme/build';
import {Incident} from '../..';
import mockData from './filteredData.test.json';
import {render, act, fireEvent} from '@testing-library/react';
import '@testing-library/jest-dom';


describe('Incident component testing', () => {
    let wrapper;

    beforeEach(() => {
        fetch.resetMocks();
    });

    afterEach(() => {
        wrapper.unmount();
    });

    it('renders successfully', () => {
        wrapper = shallow(<Incident />);
        expect(wrapper).toHaveLength(1);
    });

    it('returns mocked data', async () => {
        fetch.mockImplementation(async () => {
            return Promise.resolve({
                ok: true,
                json: () => Promise.resolve(mockData)
            });
        });
        await act(async () => {
            wrapper = render(<Incident />);
            const input = wrapper.getByLabelText('Search incident by ID');
            fireEvent.change(input, {target: {value: 'INC123'}});
        });
        await act(async () => {
            await fireEvent.click(wrapper.getByText('Search'));
        });

        expect(wrapper).toMatchSnapshot();
    });

    it('returns error', async () => {
        fetch.mockImplementation(async () => {
            return Promise.resolve({
                ok: false,
                json: () => Promise.resolve([])
            });
        });
        await act(async () => {
            wrapper = render(<Incident />);
            const input = wrapper.getByLabelText('Search incident by ID');
            fireEvent.change(input, {target: {value: 'INC123'}});
        });
        await act(async () => {
            await fireEvent.click(wrapper.getByText('Search'));
        });
        expect(wrapper.getByText('Failed to retrieve data. Try refreshing the page. If the problem persists, please message #opxhub-support or fill out our Feedback form.')).toBeInTheDocument();
        expect(wrapper).toMatchSnapshot();
    });
});