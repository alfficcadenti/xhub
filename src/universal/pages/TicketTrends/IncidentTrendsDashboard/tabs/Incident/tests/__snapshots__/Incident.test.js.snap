// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Incident component testing renders Error message when API returns error 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="inc-form-container"
        >
          <div
            class="inc-search-input form-group floating-label not-empty"
          >
            <input
              aria-invalid="false"
              aria-label="Search incident by ID"
              class="form-control"
              id="searchInput"
              name="searchInput"
              placeholder="Search incident by ID"
              type="text"
              value="INC123"
            />
            <label
              class=""
              for="searchInput"
            >
              Search incident by ID
            </label>
          </div>
          <button
            class="btn btn-primary search-btn"
            type="button"
          >
            Search
          </button>
        </div>
        <div>
          <div
            class="loading-container "
          >
            <div
              aria-live="polite"
              class="Alert border--default in loading-alert"
              role="alert"
            >
              <div
                class="Alert__flair bg--color-bg-active"
              />
              <span
                class="SVGIcon SVGIcon--24px Alert__svg inline-svg text-muted"
              >
                <svg
                  data-id="SVG_ALERT_WARNING__24"
                  focusable="false"
                  height="24"
                  viewBox="0 0 24 24"
                  width="24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    fill="none"
                    fill-rule="evenodd"
                  >
                    <path
                      d="M13 17.1a1 1 0 1 1-2 0 1 1 0 0 1 2 0"
                      fill="currentColor"
                      stroke="none"
                    />
                    <path
                      d="M10.6 4.5a1.6 1.6 0 0 1 2.8 0L21 17.7a1.6 1.6 0 0 1-1.4 2.5H4.4A1.6 1.6 0 0 1 3 17.7l7.6-13.2zM12 9V14"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                    />
                  </g>
                </svg>
              </span>
              <p>
                Failed to retrieve data. Try refreshing the page. If the problem persists, please message #opxhub-support or fill out our Feedback form.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="inc-form-container"
      >
        <div
          class="inc-search-input form-group floating-label not-empty"
        >
          <input
            aria-invalid="false"
            aria-label="Search incident by ID"
            class="form-control"
            id="searchInput"
            name="searchInput"
            placeholder="Search incident by ID"
            type="text"
            value="INC123"
          />
          <label
            class=""
            for="searchInput"
          >
            Search incident by ID
          </label>
        </div>
        <button
          class="btn btn-primary search-btn"
          type="button"
        >
          Search
        </button>
      </div>
      <div>
        <div
          class="loading-container "
        >
          <div
            aria-live="polite"
            class="Alert border--default in loading-alert"
            role="alert"
          >
            <div
              class="Alert__flair bg--color-bg-active"
            />
            <span
              class="SVGIcon SVGIcon--24px Alert__svg inline-svg text-muted"
            >
              <svg
                data-id="SVG_ALERT_WARNING__24"
                focusable="false"
                height="24"
                viewBox="0 0 24 24"
                width="24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="none"
                  fill-rule="evenodd"
                >
                  <path
                    d="M13 17.1a1 1 0 1 1-2 0 1 1 0 0 1 2 0"
                    fill="currentColor"
                    stroke="none"
                  />
                  <path
                    d="M10.6 4.5a1.6 1.6 0 0 1 2.8 0L21 17.7a1.6 1.6 0 0 1-1.4 2.5H4.4A1.6 1.6 0 0 1 3 17.7l7.6-13.2zM12 9V14"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                </g>
              </svg>
            </span>
            <p>
              Failed to retrieve data. Try refreshing the page. If the problem persists, please message #opxhub-support or fill out our Feedback form.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Incident component testing renders NO RESULTS FOUND and matches the snapshot when data is empty 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="inc-form-container"
        >
          <div
            class="inc-search-input form-group floating-label not-empty"
          >
            <input
              aria-invalid="false"
              aria-label="Search incident by ID"
              class="form-control"
              id="searchInput"
              name="searchInput"
              placeholder="Search incident by ID"
              type="text"
              value="INC123"
            />
            <label
              class=""
              for="searchInput"
            >
              Search incident by ID
            </label>
          </div>
          <button
            class="btn btn-primary search-btn"
            type="button"
          >
            Search
          </button>
        </div>
        <div>
          <div
            class="loading-container "
          >
            <div
              class="no-results-wrapper"
            >
              <span>
                NO RESULTS FOUND
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="inc-form-container"
      >
        <div
          class="inc-search-input form-group floating-label not-empty"
        >
          <input
            aria-invalid="false"
            aria-label="Search incident by ID"
            class="form-control"
            id="searchInput"
            name="searchInput"
            placeholder="Search incident by ID"
            type="text"
            value="INC123"
          />
          <label
            class=""
            for="searchInput"
          >
            Search incident by ID
          </label>
        </div>
        <button
          class="btn btn-primary search-btn"
          type="button"
        >
          Search
        </button>
      </div>
      <div>
        <div
          class="loading-container "
        >
          <div
            class="no-results-wrapper"
          >
            <span>
              NO RESULTS FOUND
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Incident component testing renders a table and matches snapshot when data is passed correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="inc-form-container"
        >
          <div
            class="inc-search-input form-group floating-label not-empty"
          >
            <input
              aria-invalid="false"
              aria-label="Search incident by ID"
              class="form-control"
              id="searchInput"
              name="searchInput"
              placeholder="Search incident by ID"
              type="text"
              value="INC123"
            />
            <label
              class=""
              for="searchInput"
            >
              Search incident by ID
            </label>
          </div>
          <button
            class="btn btn-primary search-btn"
            type="button"
          >
            Search
          </button>
        </div>
        <div>
          <div
            class="loading-container "
          >
            <div
              class="inc-details-container"
            >
              <div
                id="shortDetails"
              >
                <div
                  class="inc-row"
                >
                  <div
                    class="inc-label"
                  >
                    ID
                  </div>
                  <div
                    class="inc-value"
                  >
                    INC5997738
                  </div>
                </div>
                <div
                  class="inc-row"
                >
                  <div
                    class="inc-label"
                  >
                    Priority
                  </div>
                  <div
                    class="inc-value"
                  >
                    1-Critical
                  </div>
                </div>
                <div
                  class="inc-row"
                >
                  <div
                    class="inc-label"
                  >
                    Started
                  </div>
                  <div
                    class="inc-value"
                  >
                    2021-11-10 07:36
                  </div>
                </div>
                <div
                  class="inc-row"
                >
                  <div
                    class="inc-label"
                  >
                    Booking Impact
                  </div>
                  <div
                    class="inc-value"
                  >
                    Severe Booking Impact
                  </div>
                </div>
                <div
                  class="inc-row"
                >
                  <div
                    class="inc-label"
                  >
                    TTD
                  </div>
                  <div
                    class="inc-value"
                  >
                    11m
                  </div>
                </div>
                <div
                  class="inc-row"
                >
                  <div
                    class="inc-label"
                  >
                    Environment
                  </div>
                  <div
                    class="inc-value"
                  >
                    Production
                  </div>
                </div>
                <div
                  class="inc-row"
                >
                  <div
                    class="inc-label"
                  >
                    TTK
                  </div>
                  <div
                    class="inc-value"
                  >
                    6m
                  </div>
                </div>
                <div
                  class="inc-row"
                >
                  <div
                    class="inc-label"
                  >
                    RC Owner
                  </div>
                  <div
                    class="inc-value"
                  >
                    Egencia - Payments
                  </div>
                </div>
                <div
                  class="inc-row"
                >
                  <div
                    class="inc-label"
                  >
                    TTF
                  </div>
                  <div
                    class="inc-value"
                  >
                    0m
                  </div>
                </div>
                <div
                  class="inc-row"
                >
                  <div
                    class="inc-label"
                  >
                    L1
                  </div>
                  <div
                    class="inc-value"
                  >
                    Egencia - Consolidated
                  </div>
                </div>
                <div
                  class="inc-row"
                >
                  <div
                    class="inc-label"
                  >
                    TTR
                  </div>
                  <div
                    class="inc-value"
                  >
                    17m
                  </div>
                </div>
              </div>
              <div
                id="longDetails"
              >
                <div
                  class="inc-row-long"
                >
                  <div
                    class="inc-label-long"
                  >
                    Description
                  </div>
                  <div
                    class="inc-value-long"
                  >
                    Egencia NA degraded - created by zhroberts from #noc-ege-commands-room
                  </div>
                </div>
                <div
                  class="inc-row-long"
                >
                  <div
                    class="inc-label-long"
                  >
                    Executive Summary
                  </div>
                  <div
                    class="inc-value-long"
                  >
                    -
                  </div>
                </div>
                <div
                  class="inc-row-long"
                >
                  <div
                    class="inc-label-long"
                  >
                    Resolution Notes
                  </div>
                  <div
                    class="inc-value-long"
                  >
                    Service was restored without any technical intervention by support team.  Egencia Payment Team will lead root cause investigation.
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="inc-form-container"
      >
        <div
          class="inc-search-input form-group floating-label not-empty"
        >
          <input
            aria-invalid="false"
            aria-label="Search incident by ID"
            class="form-control"
            id="searchInput"
            name="searchInput"
            placeholder="Search incident by ID"
            type="text"
            value="INC123"
          />
          <label
            class=""
            for="searchInput"
          >
            Search incident by ID
          </label>
        </div>
        <button
          class="btn btn-primary search-btn"
          type="button"
        >
          Search
        </button>
      </div>
      <div>
        <div
          class="loading-container "
        >
          <div
            class="inc-details-container"
          >
            <div
              id="shortDetails"
            >
              <div
                class="inc-row"
              >
                <div
                  class="inc-label"
                >
                  ID
                </div>
                <div
                  class="inc-value"
                >
                  INC5997738
                </div>
              </div>
              <div
                class="inc-row"
              >
                <div
                  class="inc-label"
                >
                  Priority
                </div>
                <div
                  class="inc-value"
                >
                  1-Critical
                </div>
              </div>
              <div
                class="inc-row"
              >
                <div
                  class="inc-label"
                >
                  Started
                </div>
                <div
                  class="inc-value"
                >
                  2021-11-10 07:36
                </div>
              </div>
              <div
                class="inc-row"
              >
                <div
                  class="inc-label"
                >
                  Booking Impact
                </div>
                <div
                  class="inc-value"
                >
                  Severe Booking Impact
                </div>
              </div>
              <div
                class="inc-row"
              >
                <div
                  class="inc-label"
                >
                  TTD
                </div>
                <div
                  class="inc-value"
                >
                  11m
                </div>
              </div>
              <div
                class="inc-row"
              >
                <div
                  class="inc-label"
                >
                  Environment
                </div>
                <div
                  class="inc-value"
                >
                  Production
                </div>
              </div>
              <div
                class="inc-row"
              >
                <div
                  class="inc-label"
                >
                  TTK
                </div>
                <div
                  class="inc-value"
                >
                  6m
                </div>
              </div>
              <div
                class="inc-row"
              >
                <div
                  class="inc-label"
                >
                  RC Owner
                </div>
                <div
                  class="inc-value"
                >
                  Egencia - Payments
                </div>
              </div>
              <div
                class="inc-row"
              >
                <div
                  class="inc-label"
                >
                  TTF
                </div>
                <div
                  class="inc-value"
                >
                  0m
                </div>
              </div>
              <div
                class="inc-row"
              >
                <div
                  class="inc-label"
                >
                  L1
                </div>
                <div
                  class="inc-value"
                >
                  Egencia - Consolidated
                </div>
              </div>
              <div
                class="inc-row"
              >
                <div
                  class="inc-label"
                >
                  TTR
                </div>
                <div
                  class="inc-value"
                >
                  17m
                </div>
              </div>
            </div>
            <div
              id="longDetails"
            >
              <div
                class="inc-row-long"
              >
                <div
                  class="inc-label-long"
                >
                  Description
                </div>
                <div
                  class="inc-value-long"
                >
                  Egencia NA degraded - created by zhroberts from #noc-ege-commands-room
                </div>
              </div>
              <div
                class="inc-row-long"
              >
                <div
                  class="inc-label-long"
                >
                  Executive Summary
                </div>
                <div
                  class="inc-value-long"
                >
                  -
                </div>
              </div>
              <div
                class="inc-row-long"
              >
                <div
                  class="inc-label-long"
                >
                  Resolution Notes
                </div>
                <div
                  class="inc-value-long"
                >
                  Service was restored without any technical intervention by support team.  Egencia Payment Team will lead root cause investigation.
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
